Kernel/ThreadBlockers.cpp

```diff
@@ -436,6 +441,13 @@ void Thread::SelectBlocker::was_unblocked(bool did_timeout)
     }
 }
 
+RefPtr<Thread> Thread::SelectBlocker::pick_donate_target() const
+{
+    if (m_fds.is_empty())
+        return nullptr;
+    return m_fds[0].description->file().likely_peer_thread();
```

Comment:
Why the first one?
Why not the second one, or the most starved one, or the least scheduled one, or why not just distribute equally?

Reply:
(that is to say, please at least describe your reasoning for picking that specific fd as a comment above that, so people won't be stuck thinking "is there a specific reason" and will just see the specific reason)